CHANGELOG.latest.md

```diff
@@ -1,29 +1,44 @@
-### New Features
-* Add `StoreProduct.pricePerYear` (#2462) via NachoSoto (@NachoSoto)
-
 ### Bugfixes
-* `HTTPClient`: don't assume error responses are JSON (#2529) via NachoSoto (@NachoSoto)
-* `OfferingsManager`: return `Offerings` from new disk cache when server is down (#2495) via NachoSoto (@NachoSoto)
-* `OfferingsManager`: don't consider timeouts as configuration errors (#2493) via NachoSoto (@NachoSoto)
-
+* `AppleReceipt.debugDescription`: don't pretty-print JSON (#2564) via NachoSoto (@NachoSoto)
+* `SK2StoreProduct`: fix crash on iOS 12 (#2565) via NachoSoto (@NachoSoto)
+* `GetCustomerInfo` posts receipts if there are pending transactions (#2533) via NachoSoto (@NachoSoto)
+* `Offline Entitlements`: use offline-computed `CustomerInfo` when server is down (#2368) via NachoSoto (@NachoSoto)
```

Comment:
Hmm I don't think this is a bugfix... I would either make put in in `Other changes` or as a new feature... To me it feels like a new feature and make it a minor, even though there are no API changes. Thoughts?